### ISSUE TITLE
No output when downloading live streams from youtube with --quiet and --progress

### ISSUE THREAD

**Issue by harumando** [yt-dlp/yt-dlp#5679]
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I remove or skip any mandatory\* field

### Checklist

- [X] I'm reporting a bug unrelated to a specific site
- [X] I've verified that I'm running yt-dlp version **2022.11.11** ([update instructions](https://github.com/yt-dlp/yt-dlp#update)) or later (specify commit)
- [X] I've checked that all provided URLs are playable in a browser with the same IP and same login details
- [X] I've checked that all URLs and arguments with special characters are [properly quoted or escaped](https://github.com/yt-dlp/yt-dlp/wiki/FAQ#video-url-contains-an-ampersand--and-im-getting-some-strange-output-1-2839-or-v-is-not-recognized-as-an-internal-or-external-command)
- [X] I've searched the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)

### Provide a description that is worded well enough to be understood

There is no output when downloading youtube live streams with `--quiet` and `--progress` flags. The live stream is still downloading but there is no output (progress) in the terminal. When downloading regular videos from youtube the same way, everything works normally. When downloading live streams with flag `--live-from-start` the program prints output as intended.

Also a little question if you don't mind. When running the program with the `--newline` flag it outputs the progress with `1: ` and `2: ` prefixes. What are those and is there a way to get only one _total_ progress? 
```
yt-dlp --quiet --progress --live-from-start --no-warnings --newline https://www.youtube.com/watch?v=jfKfPfyJRdk

2: [download]    1.00KiB at    1.38KiB/s (00:00:00) (frag 0/1209118)
2: [download]    3.00KiB at    4.14KiB/s (00:00:00) (frag 0/1209118)
2: [download]    7.00KiB at    9.65KiB/s (00:00:00) (frag 0/1209118)
2: [download]   15.00KiB at   20.65KiB/s (00:00:00) (frag 0/1209118)
2: [download]   31.00KiB at   42.62KiB/s (00:00:00) (frag 0/1209118)
2: [download]   63.00KiB at   85.73KiB/s (00:00:00) (frag 0/1209118)
2: [download]   80.41KiB at  108.28KiB/s (00:00:00) (frag 0/1209118)
2: [download]   80.41KiB at  106.66KiB/s (00:00:00) (frag 1/1209118)
2: [download]   81.41KiB at   23.19KiB/s (00:00:00) (frag 1/1209118)
2: [download]   83.41KiB at   68.79KiB/s (00:00:00) (frag 1/1209118)
2: [download]   87.41KiB at  158.46KiB/s (00:00:00) (frag 1/1209118)
2: [download]   95.41KiB at  333.80KiB/s (00:00:00) (frag 1/1209118)
2: [download]  111.41KiB at  678.09KiB/s (00:00:00) (frag 1/1209118)
1: [download]    1.00KiB at    1.25KiB/s (00:00:00) (frag 0/1209118)
1: [download]    3.00KiB at    3.74KiB/s (00:00:00) (frag 0/1209118)
1: [download]    7.00KiB at    8.73KiB/s (00:00:00) (frag 0/1209118)

<...etc...>
```

### Provide verbose output that clearly demonstrates the problem

- [X] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [X] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

### Complete Verbose Output

```shell
[debug] Command-line config: ['-vU', '--quiet', '--progress', 'https://www.youtube.com/watch?v=jfKfPfyJRdk']
[debug] Encodings: locale UTF-8, fs utf-8, pref UTF-8, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version 2022.11.11 [8b64402] (zip)
[debug] Python 3.10.6 (CPython x86_64 64bit) - Linux-5.15.0-53-generic-x86_64-with-glibc2.35 (OpenSSL 3.0.2 15 Mar 2022, glibc 2.35)
[debug] exe versions: ffmpeg 4.4.2 (setts), ffprobe 4.4.2, rtmpdump 2.4
[debug] Optional libraries: Cryptodome-3.11.0, brotli-1.0.9, certifi-2020.06.20, mutagen-1.45.1, pyxattr-0.7.2, secretstorage-3.3.1, sqlite3-2.6.0, websockets-9.1
[debug] Proxy map: {}
[debug] Loaded 1723 extractors
[debug] Fetching release info: https://api.github.com/repos/yt-dlp/yt-dlp/releases/latest
Latest version: 2022.11.11, Current version: 2022.11.11
yt-dlp is up to date (2022.11.11)
[debug] [youtube] Extracting URL: https://www.youtube.com/watch?v=jfKfPfyJRdk
[youtube] jfKfPfyJRdk: Downloading webpage
[youtube] jfKfPfyJRdk: Downloading android player API JSON
[youtube] jfKfPfyJRdk: Downloading m3u8 information
[youtube] jfKfPfyJRdk: Downloading m3u8 information
[debug] Sort order given by extractor: quality, res, fps, hdr:12, source, vcodec:vp9.2, channels, acodec, lang, proto
[debug] Formats sorted by: hasvid, ie_pref, quality, res, fps, hdr:12(7), source, vcodec:vp9.2(10), channels, acodec, lang, proto, filesize, fs_approx, tbr, vbr, abr, asr, vext, aext, hasaud, id
[debug] Default format spec: best/bestvideo+bestaudio
[info] jfKfPfyJRdk: Downloading 1 format(s): 96
[debug] Invoking ffmpeg downloader on "https://manifest.googlevideo.com/api/manifest/hls_playlist/expire/<removed>/playlist/index.m3u8"
[download] Destination: lofi hip hop radio - beats to relax⧸study to 2022-12-01 00_03 [jfKfPfyJRdk].mp4
[debug] ffmpeg command line: ffmpeg -y -loglevel quiet -headers 'User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/94.0.4606.54 Safari/537.36
Accept: text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8
Accept-Language: en-us,en;q=0.5
Sec-Fetch-Mode: navigate
' -i https://manifest.googlevideo.com/api/manifest/hls_playlist/expire/<removed>/playlist/index.m3u8 -c copy -f mpegts 'file:lofi hip hop radio - beats to relax⧸study to 2022-12-01 00_03 [jfKfPfyJRdk].mp4.part'

^C

[ffmpeg] Interrupted by user
[ffmpeg] Downloaded 1820968 bytes
[debug] ffprobe command line: ffprobe -hide_banner -show_format -show_streams -print_format json 'file:lofi hip hop radio - beats to relax⧸study to 2022-12-01 00_03 [jfKfPfyJRdk].mp4'
[FixupM3u8] Fixing MPEG-TS in MP4 container of "lofi hip hop radio - beats to relax⧸study to 2022-12-01 00_03 [jfKfPfyJRdk].mp4"
[debug] ffmpeg command line: ffmpeg -y -loglevel repeat+info -i 'file:lofi hip hop radio - beats to relax⧸study to 2022-12-01 00_03 [jfKfPfyJRdk].mp4' -map 0 -dn -ignore_unknown -c copy -f mp4 -bsf:a aac_adtstoasc -movflags +faststart 'file:lofi hip hop radio - beats to relax⧸study to 2022-12-01 00_03 [jfKfPfyJRdk].temp.mp4'
```

**Comment by Grub4K** [comment]
As you can see, when `--live-from-start` is not used the stream is downloaded using `ffmpeg`.
`--quiet` will silence all output from `ffmpeg` and since progress updates for `ffmpeg` have not yet been implemented, `--progress` will have no effect.
Wait until this has been implemented, then it should work.

This is a duplicate of the other progress requests (eg #1720)

- https://github.com/yt-dlp/yt-dlp/pull/2475

Regarding your question, it downloads the audio and video in parallel.
Generally multiline progress will look like this (one progress refers to audio and one to video):
```
[download]    1.16MiB at   34.22KiB/s (00:00:06) (frag 3/1209536)
[download]  320.40KiB at  100.03KiB/s (00:00:05) (frag 4/1209536)
```
Since the lines are fixed in place, you can discern which is which. If they are printed with `--newline` though, it could be either which is being output. To circumvent this, an index (`1: `, `2: `) is prepended to be able to discern which progress is which.

Does this clarify it?

**Comment by harumando** [comment]
@Grub4K It does clarify, yes, thank you. Sorry for the duplicate.
